src/eradiate/scenes/bsdfs/_rtls.py

```diff
@@ -0,0 +1,184 @@
+from __future__ import annotations
+
+import attrs
+import pinttr
+import mitsuba as mi
+
+from ._core import BSDF
+from ..core import traverse
+from ..spectra import Spectrum, spectrum_factory
+from ... import validators
+from ...units import unit_context_config as ucc
+from ...attrs import documented, parse_docs
+from ...kernel import TypeIdLookupStrategy, UpdateParameter
+
+
+@parse_docs
+@attrs.define(eq=False, slots=False)
+class RTLSBSDF(BSDF):
+    """
+    RTLS BSDF [``rtls``].
+
+
+
+    See Also
+    --------
+    :ref:`plugin-bsdf-rtls`
+
+    Implements the Ross-Thick, Li-Sparse (RTLS) BRDF
+    :cite:`BU-MODISBRDFAlbedoProductATBD1999`.
+
+    The RTLS class exposes the 3 parameters :math:`f_{iso}`, :math:`f_{vol}` and
+    :math:`f_{geo}` configuring the linear combination the wreights of the
+    isometric, volumetric and geometric kernel values in the BSDF definition
+    respectively.
```

Comment:
Suggestion to better comply with the Numpydoc style and remove redundant information:
```suggestion
    RTLS BSDF [``rtls``].

    This class implements the RossThick-LiSparse (RTLS) BRDF as described
    by the MODIS BRDF/Albedo Product ATBD :cite:`BU-MODISBRDFAlbedoProductATBD1999`.

    See Also
    --------
    :ref:`plugin-bsdf-rtls`
```
Also, the other parameters (h, r, b) are also configurable; this